lib/rubocop/cop/style/each_with_object.rb

```diff
@@ -90,6 +96,14 @@ def first_argument_returned?(args, return_value)
 
           accumulator_var == return_var
         end
+
+        def return_value_occupies_whole_line?(node, return_value)
+          node.body.source.lines.last =~ /\s*#{return_value.source}\s*/
```

Comment:
Would be nice if this could compare ranges rather than use a regular expression, which can lead to problems with inline comments, etc.